app/src/main/java/com/dineout/code/hall/Adapter.java

```diff
@@ -244,17 +244,17 @@ public void bindView(final int position) {
             TableID.setText("Id: ");
             Capacity.setText(tables.get(position).getCapacity());
             TableCapacity.setText("Capacity: ");
-            if (tables.get(position).getStatus().equals("Booked")) {
+            if ("Booked".equals(tables.get(position).getStatus())) {
```

Comment:
Once again, interesting fix for all of the possible null pointer exceptions, nice